src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,74 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageCount = [];
+
+  for (let i = 1; i <= Math.ceil(total / perPage); i++) {
+    pageCount.push(i);
+  }
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage <= 1}
+          onClick={
+            currentPage > 1 ? () => onPageChange(currentPage - 1) : undefined
```

Comment:
Redudant check, if it will be disabled

Reply:
Without this condition, the tests do not pass. I was recommended to do it this way in the chat.